js/scripts.js

```diff
@@ -24,16 +27,19 @@ function Table() {
   this.headsUp = false;
   this.atBat;
   this.betOccurred = false;
-  this.round = ""
-  this.pot = 0
+  this.round = "";
+  this.pot = 0;
   this.suits = [
-    'clubs',
-    'diamonds',
     'hearts',
-    'spades'
+    'spades',
+    'diamonds',
+    'clubs'
   ];
+  // this.ranks = [
+  //   '2','3','4','5','6','7','8','9','10','jack','queen','king','ace'
+  // ];
   this.ranks = [
-    '2','3','4','5','6','7','8','9','10','jack','queen','king','ace'
+    'ace','two','three','four','five','six','seven','eight','nine','ten','jack','queen','king'
```

Comment:
This will break the straightsList on line 79, make sure to fix after merge.

Reply:
Okay, noted! Thanks!